qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -202,8 +212,8 @@ def test_rmdir_non_existing_dir(self):
         """
         Test that rmdir does not delete a non existing directory
         """
-        rmdir_output = self.get_cephfs_shell_cmd_error("rmdir test_dir")
-        log.info("rmdir error output:\n{}".format(rmdir_output))
+        rmdir_output = self.negtest_cephfs_shell_cmd(
+            cmd="rmdir test_dir", errmsg='no such file or directory')
```

Comment:
We need not check error message. Based on discussion [here](https://github.com/ceph/ceph/pull/31633#discussion_r355448961)

Reply:
Right. Didn't see that next line will actually perform a stat to confirm.